muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/containers/user_credentials.tsx

```diff
@@ -0,0 +1,13 @@
+import Notifications from '../components/base/notifications';
+import Body from '../components/credentials/body';
+import * as React from 'react';
+import '~/sass/util/base.scss';
+
```

Comment:
can we change the name to user-credentials.tsx?

If we have been using that naming convention.